app/config.go

```diff
@@ -319,32 +336,37 @@ func initFiles(address sdk.Address, password string, datadir string) string {
 		fmt.Println("Initializing keybase: enter validator passphrase")
 		password = Credentials()
 		if password == "" {
-			panic("you must have a validator account password")
+			fmt.Println("you must have a validator account password")
+			os.Exit(1)
 		}
 		err := newKeybase(password)
 		if err != nil {
-			panic(err)
+			fmt.Println(err)
+			os.Exit(1)
 		}
 	}
 	if _, err := os.Stat(datadir + FS + GlobalConfig.TendermintConfig.PrivValidatorKey); err != nil {
 		if os.IsNotExist(err) {
 			password = privValKey(address, password)
 		} else {
-			panic(err)
+			fmt.Println(err)
+			os.Exit(1)
```

Comment:
```suggestion
                     log.Fatal(err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work